themes/psh-docs/layouts/shortcodes/guides/gatsby/headless-project.md

```diff
@@ -19,4 +19,4 @@ When deploying a single app project [such as Gatsby](/guides/gatsby/deploy.html)
 the repository needs three configuration files that describe its infrastructure, described below in detail.
 For [multi-app projects](/create-apps/multi-app/_index.md),
 two of those files remain in the project root and are shared between Gatsby and {{ .Get "name" }}.
-Each app keeps its own app configuration file (`.platform.app.yaml`) in its subdirectory.
+Each app keeps its own app configuration file (`{{ partial "vendor/configfile" (dict "context" . "config" "services") }}`) in its subdirectory.
```

Comment:
on the previous version we were referencing `.platform.app.yaml` on the change we appear to be referencing the vendor's services configuration file?